### ISSUE TITLE
In VSCode, how do I hide the 'Run Cell | Run Next Cell | Run Above' buttons on top of every code chunk?

### ISSUE THREAD

**Issue by faurehu** [quarto-dev/quarto#68]
I keep accidentally clicking Run Above, which is both time consuming and also overwrites some of my variables.

**Comment by jjallaire** [comment]
There isn't currently a way to hide these (will keep this bug open as a reminder to see about adding an option for this)

**Comment by heavywatal** [comment]
We can hide them by setting `"editor.codeLens": false` globally or in a language-specific scope:
```json
"[quarto]": {
  "editor.codeLens": false,
},
```

**Comment by faurehu** [comment]
Beautiful, thanks! Hopefully we can customise in the future?

**Comment by jjallaire** [comment]
Yes, we'll add a setting for this (although it would really just be sugar over the setting described above, so potentially also well handled in documentation)
